README.md

```diff
@@ -1,3 +1,5 @@
+#####The following is prior to change and does NOT fully reflect current state.
```

Comment:
Cool. The documentation could use a cleanup. 

There's no need to explain what `REST` is and why it's useful. The documentation should focus on the BPI rest service and it's use.

It would be really nice with instructions on how the get the service running, i.e. what the requirements are and how to set it up for development (e.g. using `Docker` or similar means) and production.

Reply:
@rimi-itk Thank you for the input. Can you create an issue on http://platform.dandigbib.org describing your wishes to the README and or interactive API docs (like Swagger).

Reply:
@guddo I've outlines my ideas for improved setup/development instructions on https://platform.dandigbib.org/issues/4061.

Interactive API docs would be really useful as well. Does the project currently provide this?

Reply:
No, it does not @rimi-itk - please create an issue for this as well :-)